frontend/src/components/ChoicesFilterSection.tsx

```diff
@@ -1,40 +1,77 @@
 import React from 'react';
 
-import { Typography, FormControlLabel, Checkbox, Box } from '@material-ui/core';
-import { CheckCircle, CheckCircleOutline } from '@material-ui/icons';
 import { TitledSection } from 'src/components';
+import { Typography, FormControlLabel, Checkbox, Box } from '@material-ui/core';
+import {
+  CheckCircle,
+  CheckCircleOutline,
+  CheckBox,
+  CheckBoxOutlineBlank,
+} from '@material-ui/icons';
 
 interface Props {
   title: string;
   value: string;
   choices: Record<string, string>;
+  multipleChoice: boolean;
   onChange: (value: string) => void;
 }
 
-const ChoicesFilterSection = ({ title, value, choices, onChange }: Props) => {
+const ChoicesFilterSection = ({
+  title,
+  value,
+  choices,
+  multipleChoice,
```

Comment:
Very minor: a default value for `multipleChoice` may improve the usability of this component.  
```suggestion
  multipleChoice = false,
```